Style.css

```diff
@@ -0,0 +1,119 @@
+html, body {
```

Comment:
Hi, @cancelei!! 👋
- Well done! Here you have made a good suggestion with reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better not to specify 2 times.......". 
- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.